packages/fxa-settings/src/pages/CompleteResetPassword/index.test.tsx

```diff
@@ -25,32 +28,56 @@ const alertBarInfo = {
   error: jest.fn(),
 } as unknown as AlertBarInfo;
 
-describe('CompleteResetPassword', () => {
+describe('CompleteResetPassword page', () => {
+  // TODO: enable l10n tests when they've been updated to handle embedded tags in ftl strings
+  // TODO: in FXA-6461
```

Comment:
Thanks for always adding these TODOs. It'll make life a lot easier whenever we get to these, which I hope will be soon...